### DISCUSSION TITLE
Problem with External IPs except 127.0.0.1

### DISCUSSION THREAD

**Discussion by hamid-m331** [Abbas-gheydi/radotp#3] [General]
Hello dear and thank you for this practical project

I found login problem with the interface IP of the VM while entering to the web page.
Would you mind explain the reason and how I can handle this issue ?

**Comment by abbas-gheydi** [comment]
thnx dear Hamid, I think this link can help you:
https://github.com/Abbas-gheydi/radotp/discussions/2#discussioncomment-6096140)
